types/rehype-react/index.d.ts

```diff
@@ -8,6 +8,7 @@ import * as React from "react";
 
 interface Options {
     createElement: typeof React.createElement;
+    Fragment: React.ComponentType<{ children?: React.ReactNode }>;
```

Comment:
What about using React.Fragment type directly?

Reply:
That's a good point. Probably would have been preferable.